pymemcache/test/test_client.py

```diff
@@ -902,6 +924,49 @@ def test_socket_connect_closes_on_failure(self):
         assert socket_module.sockets[0].connections == []
         assert socket_module.sockets[0].closed
 
+    def test_socket_flush_on_reconnect(self):
+
+        server = ("example.com", 11211)
+        timeout = 2
+        connect_timeout = 3
```

Comment:
I'm not sure breaking these values out into variables improves readability. I would probably just write them as keyword values when constructing `Client`:

```python
        client = Client(("example.com", 11211), connect_timeout=3,
                        timeout=2,
                        socket_module=mock_permanent_socket,
                        flush_on_reconnect=True)
```

Reply:
done